libqtile/scripts/migrate.py

```diff
@@ -162,7 +162,7 @@ def do_migrate(args):
 
     for m in MIGRATIONS:
         q = bowler.Query(config_dir)
-        m(q).execute(interactive=args.interactive, write=True)
+        m(q).execute(interactive=not args.non_interactive, write=True)
```

Comment:
Isn't the double negation wrong if store_false?

Reply:
Yeah, derp :). It should be store_true still.